app/viewmodels/CYASummaryListHelper.scala

```diff
@@ -445,4 +445,40 @@ class CYASummaryListHelper {
     } else None
   }
 
+  def buildDefermentSummaryList(answer: UserAnswers)(implicit messages: Messages): Option[CYASummaryList] = {
+    val paymentInformationSummaryListRow: Option[Seq[SummaryListRow]] = answer.get(DefermentPage) map { deferment =>
+      val payingByDeferment = if (deferment) "Yes" else "No"
+      Seq(
+        SummaryListRow(
+          key = Key(
+            content = Text(messages("cya.payingByDeferment")),
+            classes = "govuk-!-width-two-thirds govuk-!-padding-top-0"
+          ),
+          value = Value(
+            content = HtmlContent(payingByDeferment),
+            classes = "govuk-!-padding-top-0"
```

Comment:
As this is the only entry for this Summary List Row - padding is not needed

Please remove whole class line